rules-tests/Php80/Rector/NotIdentical/StrContainsRector/Fixture/offset_variable_equal_multibyte_strpos.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\NotIdentical\StrContainsRector\Fixture;
+
+class OffsetVariableMultibyteStrpos
+{
+    public function run()
+    {
+        $offset = 1;
+        $isMatch = mb_strpos('abc', 'a', $offset) != false;
```

Comment:
real multibyte char test is needed

Reply:
For all my multi byte tests i added `happy 😊 euro €`